src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ListViewInsertionMarkTests.cs

```diff
@@ -522,9 +521,9 @@ public unsafe void ListViewInsertionMark_Index_GetInsertMarkWithColor_Success(in
             // Set same.
             Assert.NotEqual(IntPtr.Zero, control.Handle);
             control.InsertionMark.Index = 0;
-            var insertMark = new ComCtl32.LVINSERTMARK
+            var insertMark = new LVINSERTMARK
```

Comment:
When you change code it would be good to flip to target type new to match our desired pattern

```suggestion
            LVINSERTMARK insertMark = new()
```

Reply:
Don't sweat changing everything in the file, just what you touch.

Reply:
It was a small file, so I did everything.